maint/local_python/binding_f08.py

```diff
@@ -1581,8 +1581,11 @@ def get_int_type(fortran_type):
     large_map = G.MAPS['BIG_F08_KIND_MAP']
     for kind in small_map:
         if kind == 'POLYFUNCTION':
-            # TODO: potentially tricky. Might be easier to treat individual function case by case.
-            G.real_poly_kinds[kind] = 1
+            # Currently there are only two: MPI_User_function, MPI_Datarep_conversion_function,
+            # both contains parameter POLYXFER_NUM_ELEM. However, Fortran is not able to
+            # differentiate generic interface based on different function signature. Disable
+            # for now.
+            pass
```

Comment:
Just to help my understanding: the end result is that we don't generate a polymorphic interface for these callbacks. Instead, there are explicit small and large interfaces for use in the small and large registration functions, respectively. It is up to the user to use the right one.

Reply:
Yes, your understanding is correct.